zerver/tests/test_event_system.py

```diff
@@ -977,14 +977,14 @@ def test_queries(self) -> None:
             with mock.patch("zerver.lib.events.always_want") as want_mock:
                 fetch_initial_state_data(user)
 
-        self.assert_length(queries, 31)
+        self.assert_length(queries, 32)
```

Comment:
There is an increase in one query because now, we are checking for `tutorial_status` after `NON_INTRO_HOTSPOTS` which caused https://github.com/zulip/zulip/pull/17557/commits/5137ca939491864364174ad6695dc01906b44c62#diff-37b1b8cd541dd2f2830f3ffeddfd29df77efb005849fe6fd2277e8dcd608604aR82 to execute.

Reply:
We can avoid this query by setting `TUTORIAL_ENABLED = False`.
Should we do it?

Reply:
Generally no -- the goal in a test like this is to measure what work is done, not avoid numbers going up.